router/router.go

```diff
@@ -261,3 +263,41 @@ func SupportCORS(handler http.Handler) http.Handler {
 		AllowedHeaders: []string{"Origin", "X-Requested-With", "Content-Type", "Accept"}})
 	return c.Handler(handler)
 }
+
+type defReq struct {
+	Ext defExt `json:"ext"`
+}
+type defExt struct {
+	Prebid defaultAliases `json:"prebid"`
+}
+type defaultAliases struct {
+	Aliases map[string]string `json:"aliases"`
+}
+
+func readDefaultRequest(defReqConfig config.DefReqConfig) (map[string]string, []byte) {
+	defReq := &defReq{}
+	aliases := make(map[string]string)
+	if defReqConfig.Type == "file" {
+		if len(defReqConfig.FileSystem.FileName) == 0 {
+			return aliases, []byte{}
+		}
+		defReqJSON, err := ioutil.ReadFile(defReqConfig.FileSystem.FileName)
+		if err != nil {
+			glog.Errorf("error reading aliases from file %s: %v", defReqConfig.FileSystem.FileName, err)
```

Comment:
Should fail fast & crash on invalid configs, for easier debugging.

Here are some "error" cases:

1. `type` is something besides `file` or empty string. (prevents typos like "fild")
2. `type` is `file`, but `filesystem.filename` is empty/undefined.
3. `type` is `file`, but the `filesystem.filename` doesn't exist on the filesystem.